src/App.tsx

```diff
@@ -1,10 +1,61 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { MainFilter } from './types/MainFilter';
+import { getTodos } from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { Loader } from './components/Loader/Loader';
 
-const USER_ID = 0;
+const USER_ID = 9975;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorOfLoading, setErrorOfLoading] = useState('');
+  const [isTodosLoading, setIsTodosLoading] = useState(true);
+  const [filterOption, setFilterOption] = useState(MainFilter.All);
+  const [numberOfActiveTodos, numberOfCompletedTodos] = useMemo(
+    () => [
+      todos.filter(({ completed }) => !completed).length,
+      todos.filter(({ completed }) => completed).length,
+    ],
+    [todos],
+  );
+
+  const getTodosFromServer = async () => {
+    try {
+      const todosFromServer = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    } catch {
+      setErrorOfLoading('Something was wrong, during the loading...');
+    } finally {
+      setIsTodosLoading(false);
+    }
+  };
+
+  const getFilteredTodos = (filter: MainFilter, someTodos: Todo[]) => {
+    if (filter === MainFilter.Active) {
+      return someTodos.filter((todo) => !todo.completed);
+    }
+
+    if (filter === MainFilter.Completed) {
+      return someTodos.filter((todo) => todo.completed);
+    }
+
+    return someTodos;
```

Comment:
use switch/case

Reply:
Alisa, thank you for review and recommendation! I have implemented it!